src/components/pages/Offer/OfferObject.js

```diff
@@ -0,0 +1,10 @@
+export default class OfferObject {
```

Comment:
`Object` est trop générique à mon goût.

Pourquoi pas simplement Offer ?
```suggestion
export default class Offer {
```

Reply:
J'aimerai bien mettre Offer seul aussi, mais on a déjà un fichier Offer au même niveau, la page Offer.jsx.

Reply:
Du coup, je me disais qu'on pouvait peut-être le ranger dans un dossier au même niveau qui recense tout les value objects aussi.
Ça pourrait le faire ?